configure.ac

```diff
@@ -20,8 +20,9 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 
+
 AC_PREREQ([2.69])
-AC_INIT([cc-oci-runtime], [2.1.7])
+AC_INIT([cc-oci-runtime], m4_esyscmd_s([cat versions.txt  | grep cc_oci_runtime_version | cut -d'=' -f2]))
```

Comment:
I'm rather surprised this works, since conventionally `configure.ac` should be updating `versions.txt.in`. @dlespiau , @devimc - do you have any thoughts on this approach? Anything that could "bite us" later on?